.vscode/settings.json

```diff
@@ -18,5 +18,6 @@
   },
   "extensions.ignoreRecommendations": true,
   "files.autoSave": "off",
-  "debug.allowBreakpointsEverywhere": true
+  "debug.allowBreakpointsEverywhere": true,
+  "gitdoc.enabled": false
```

Comment:
gitdoc is a vscode extension that does automatic git-commits (and I think maybe push and pull as well) in real-time to do snapshots like a google doc, which would not be appropriate in this situation (It would create a very messy-looking commit history in this repo).
It's kinda weird and only makes sense in very specific situations, so I have it disabled by default on my end and it automatically added that to the `.vscode/settings.json`, which is probably good because we would definitely not want that extension to be enabled by accident.